### DISCUSSION TITLE
Show custom functions in the expression builder

### DISCUSSION THREAD

**Discussion by Miurath** [eclipse-birt/birt#1363] [Q&A] [answered]
I have written a custom function in the `inizialize()` stage of the `.rptdesign`. I know I can just type the function in the expression builder, but I'd like to see an overview of my custom functions. Is there a way to show my custom functions in the expression Builder?

Thanks and have a nice day.

**Comment by hvbtup** [comment, marked as answer]
Unfortunately not.
